modules/statistics/CallStats.js

```diff
@@ -47,7 +47,7 @@ const fabricEvent = {
     activeDeviceList: 'activeDeviceList'
 };
 
-let callStats = null;
+let callStatsBackend = null;
```

Comment:
❤️ I would even go as far as calling it "backend", since the file is called CallStats.